tests/foreman/cli/test_template.py

```diff
@@ -249,111 +249,58 @@ def test_positive_clone(self):
-        """
-
-    @stubbed
-    @tier3
-    def test_positive_export_log_to_production(self):
-        """Assert template export logs are logged to production logs
-
-        :id: 55dca9ce-18b1-4846-bae9-44a3b94a2749
-
-        :Steps:
-            1. Using hammer, Export template to destination
-
-        :expectedresults:
-            1. Assert template export task and status logged to production log
-
-        :Requirement: Take Templates out of tech preview
-
-        :CaseAutomation: NotAutomated
-
-        :CaseLevel: System
-
-        :CaseAutomation: NotAutomated
-        """
-
-    @stubbed
-    @tier3
-    def test_rake_templates_import_depreciated(self):
-        """Assert running `foreman-rake templates:import` shows depreciation
-        message
+    @tier2
+    def test_negative_import_locked_template(self, module_org, create_import_export_local_dir):
+        """Assure locked templates are not pulled from repository.
 
-        :id: 75aab2b2-fcbb-436c-924d-4897d49ee0d2
+        :id: 88e21cad-448e-45e0-add2-94493a1319c5
 
         :Steps:
-            1. Run `foreman-rake templates:import` task through
-            command from a directory having no templates to import
+            1. Using nailgun or direct API call try to import a locked template
 
         :expectedresults:
-            1. Assert task shows depreciation message of rake import is
-            depreciated
+            1. Assert locked template is not updated
 
-        :CaseImportance: Low
+        :CaseImportance: Medium
 
         :CaseAutomation: NotAutomated
         """
+        prefix = gen_string('alpha')
+        dir_path = create_import_export_local_dir
+        ssh.command('cp example_template.erb {}'.format(dir_path))
+        pre_template = entities.Template().imports(
+            data={
+                'repo': dir_path,
+                'organization_id': module_org.id,
+                'prefix': prefix,
+                'lock': True,
+            }
+        )
+        assert entities.ProvisioningTemplate().
 
-    @stubbed
-    @tier3
-    def test_rake_templates_export_depreciated(self):
-        """Assert running `foreman-rake templates:export` shows depreciation
-        message
-
-        :id: b9e30f89-1cbf-47e0-ad2d-3e120ebe0643
-
-        :Steps:
-            1. Run `foreman-rake templates:export` task through
-            command to a directory to export templates
-
-        :expectedresults:
-            1. Assert task shows depreciation message of rake export is
-            depreciated
-
-        :CaseImportance: Low
-
-        :CaseAutomation: NotAutomated
-        """
 
-    @stubbed
-    @tier3
-    def test_rake_templates_purge_depreciated(self):
-        """Assert running `foreman-rake templates:purge` shows depreciation
-        message
+    @stubbed()
+    @tier2
+    def test_positive_import_force_locked_template(self):
+        """Assure locked templates are updated from repository when `force` is
+        specified.
```

Comment:
```suggestion
        """Assure locked templates are updated from the repository when the `force` parameter is specified. 
```

Reply:
@omkarkhatavkar The PR is Draft PR - Work In Progress!

Reply:
Ohh sorry I missed that, will review later